src/tests/Common/CoreCLRTestLibrary/Vectors.cs

```diff
@@ -0,0 +1,141 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace TestLibrary
+{
+    public static class Vectors
+    {
+        public static void VectorToArray<T>(ref T[] to, Vector<T> from)
+        {
+            long tsize = Unsafe.SizeOf<T>();
+            long vsize = Unsafe.SizeOf<Vector<T>>();
+            if (to.Length * tsize != vsize)
+            {
+                throw new ArgumentException("sizeof(to) != sizeof(from)");
+            }
+            Unsafe.WriteUnaligned(ref Unsafe.As<T, byte>(ref to[0]), from);
+        }
+
+        public static void ArrayToVector<T>(ref Vector<T> to, T[] from)
```

Comment:
I don't think we're using this anywhere yet. Instead of adding this helper, can we just use the `Vector<T>` constructor that initializes from an array? ex: `Vector<T> vec = new Vector<T>(from)`.